### PR TITLE
Added more packages

### PR DESCRIPTION
**Pull request by jbednar**
Adds entries for various HoloViz.org plus other packages listed at pyviz.org that use Matplotlib or have support for Matplotlib. HoloViz packages include:

- colorcet
- datashader
- geoviews
- holoviews
- hvplot
- lumen
- panel

Non-HoloViz but relevant other packages included:

- geopandas
- networkx
- mpl-scatter-density
- glue

Note that hvPlot's Matplotlib support is still in progress (https://github.com/holoviz/hvplot/pull/653), but I don't want to add to review burden by postponing that until it's ready, which should be very soon.

Depends on the section naming from https://github.com/matplotlib/mpl-third-party/pull/94 , so if that's changed this PR will need updating.

### CODE REVIEW

**Comment by jbednar** [comment]
The failing tests are just due to #94 not being merged; would need rebasing after that (or a merge without worrying about the tests).

**Comment by jbednar** [comment]
Ok, @jklymak and @story645 et al., this one is ready for review now that #94 has been merged.

**Comment by QuLogic** [comment]
I don't understand these 'Changes from upstream' commits; can you not rebase or merge?

**Comment by jbednar** [comment]
Me either, but the diffs should already be clean for a squash merge in any case, so I don't think it matters...

**Comment by jbednar** [comment]
This should be straightforward to review; I think it's ready to merge unless someone has suggestions about the descriptions of the projects.

File: packages/mpl-scatter-density.yml
Comment (story645) [inline packages/mpl-scatter-density.yml:2]: I think this and datashader are plot types ?

File: packages/pandas.yml
Comment (story645) [inline packages/pandas.yml:5]: Tabular? data analysis

Comment (jbednar) [reply packages/pandas.yml:5]: It's both columnar and tabular, but I'm happy to call it tabular instead!

```suggestion
description: Tabular data analysis and manipulation tool providing a .plot() API for Matplotlib plotting.
```

File: packages/mpl-scatter-density.yml
Comment (jbednar) [reply packages/mpl-scatter-density.yml:2]: Yes and no, but sure!

```suggestion
section: plot types
```

File: packages/datashader.yml
Comment (jbednar) [inline packages/datashader.yml:2]:
```suggestion
section: plot types
```

**Review by story645 — APPROVED** [review]

File: packages/geoviews.yml
Comment (QuLogic) [inline packages/geoviews.yml:4]:
```suggestion
description: High-level geographic plotting built on Cartopy and Matplotlib.
```

**Review by QuLogic — COMMENTED** [review]
Also, it appears all the existing descriptions end in a period; can you add one on these?

**Comment by jbednar** [comment]
Good catch! Added trailing periods to all descriptions, including a few not otherwise touched in this PR.

**Review by QuLogic — APPROVED** [review]

**Pull request merged**
